app/jobs/sqs_worker.rb

```diff
@@ -0,0 +1,22 @@
+class SqsWorker
+  include Shoryuken::Worker
+
+  # TODO: set real queue name
+  # TODO: set auto_delete: true after testing
+  shoryuken_options queue: 'TODO-add-real-queue', body_parser: :json, auto_delete: false
```

Comment:
Do we actually need Shoryuken here? What's the advantage to using this when we have Delayed::Job?